crates/ibc/src/context/nft_transfer.rs

```diff
@@ -28,6 +28,7 @@ where
     C: IbcCommonContext,
 {
     inner: Rc<RefCell<C>>,
+    is_shielded: bool,
```

Comment:
Why do we have this new field?

Reply:
Because the sender in the IBC packet is replaced with the refund address. The context doesn't know whether the sender is the actual sender or the refund target.

Reply:
Right, makes sense, thank you